src/containers/shared/components/TransactionTable/TransactionTable.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { MouseEventHandler } from 'react'
+import { useTranslation } from 'react-i18next'
+import { TransactionTableRow } from './TransactionTableRow'
+import Loader from '../Loader'
+import { LoadMoreButton } from '../../LoadMoreButton'
+import './styles.scss'
+
+export type TransactionTableProps = React.HTMLAttributes<HTMLElement> & {
+  transactions: any[] | undefined
+  emptyMessage?: string
+  loading: boolean
+  onLoadMore: MouseEventHandler
+  hasAdditionalResults: boolean | undefined
```

Comment:
ditto

Reply:
also this is weird - seems like the `undefined` case should coalesce to `false` somewhere

Reply:
Having it optional is fine and default to false